lib/include/kframework/avm/teal/teal-fields.md

```diff
@@ -71,6 +71,10 @@ module TEAL-FIELDS
 ### `txn`/`gtxn` fields
 
 ```k
+
+  syntax TxnFieldTop ::= TxnField
+                       | TxnaFieldExt
```

Comment:
I'm confused by the naming here. What does `TxnaFieldExt` stand for?

Reply:
I'm not sure, to be honest. Maybe "extension". This was a sort that was there from before my changes. I guess there were certain fields that were not accessible in some contexts and these were the `Ext` fields. However I moved all these fields to just be normal `TxnaField`s because they needed to be accessible by the `txna`. It's probably fine to just get rid of the distinction.
Another thing about naming here, I don't really like using the name TxnFieldTop to represent TxnField + TxnaField because it seems too generic but I haven't thought of anything better yet.